serenity/qserenitybackingstore.cpp

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 The Qt Company Ltd.
```

Comment:
Why does this have a QT copyright?

Reply:
Fixed, changed all headers to LGPLv3, added a LICENSE file.

Reply:
Why LGPLv3? It would be nice if this project was the same license as serenity? @awesomekling thoughts?

Reply:
I chose a license that's used by Qt. I checked out a few other 3rd party Qt platform plugins and the generally resorted to only using GPLv3 or v3 along with v2

Reply:
Since this is part of the Serenity project, we would like for it to follow the License of the rest of the Serenity code.

Reply:
I really have no preference in this regard so I just picked whatever came first, sorry. 2-clause BSD seems to be compatible with GPL3 so why not. I replaced the code with BSD licensing.